octoprint_mrbeam/files/material_settings/materials.csv

```diff
@@ -558,7 +558,7 @@ MrB II Dreamcut x,Sign Material,#f3de8d,"1,5",100,500,1,3,0,no,
 MrB II Dreamcut x,Sign Material,#d2a477,Engrave,0-50,2000-1400,1,3,0,no,
 MrB II Dreamcut x,Sign Material,#d2a477,"1,5",100,500,1,3,0,no,
 MrB II Dreamcut x,Swiss stone pine,#bc955a,Engrave,0-20,2000-2000,1,3,0,no,tested
-MrB II Dreamcut x,Swiss stone pine,#bc955a,3,100,500,2,3,0,no,
```

Comment:
This is not assigned to any material

Reply:
We have this a lot, it uses the last defined from above, this happens if you join cells in the spreadsheet